packages/block-editor/src/components/off-canvas-editor/block-contents.js

```diff
@@ -51,12 +51,13 @@ const ListViewBlockContents = forwardRef(
 				const {
 					hasBlockMovingClientId,
 					getSelectedBlockClientId,
-					getLastInsertedBlockClientId,
+					__experimentalGetLastInsertedBlockClientId,
 				} = select( blockEditorStore );
 				return {
 					blockMovingClientId: hasBlockMovingClientId(),
 					selectedBlockInBlockEditor: getSelectedBlockClientId(),
-					lastInsertedBlockClientId: getLastInsertedBlockClientId(),
+					lastInsertedBlockClientId:
+						__experimentalGetLastInsertedBlockClientId(),
```

Comment:
Did we merge the "@wordpress/experiments" PR yet, in which case prefixing is considered like an outdated approach?

Reply:
Maybe @adamziel would know the best approach to introduce an experimental selector right now?

Reply:
Yes that's a good point. It might be worth a separate PR to make it experimental and then rebase _this_ PR.

Reply:
Due to the flux in the API since introduced, I've followed up and made this a private selector 

https://github.com/WordPress/gutenberg/pull/47638